packages/next/package.json

```diff
@@ -56,7 +56,7 @@
     "types": "tsc --declaration --emitDeclarationOnly --declarationDir dist",
     "typescript": "tsc --noEmit --declaration",
     "ncc-compiled": "ncc cache clean && taskr ncc",
-    "build-native": "npx -p @napi-rs/cli@1.2.1 napi build --platform --cargo-cwd build/swc native"
+    "build-native": "napi build --platform --cargo-cwd build/swc native"
```

Comment:
I was using `npx` so we wouldn't have to wait for all of the dependencies to install. We would need to add the `build` job as a dependency to the native build jobs with this change.

Reply:
We already depend on the `build` job it looks like, I don't think we download the cache from the build job though

Reply:
Ah, forgot we added that to get access to the `needs.build.outputs.isRelease`. We should be good then.